http-2.gemspec

```diff
@@ -4,19 +4,19 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'http/2/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "http-2"
+  spec.name          = 'http-2'
   spec.version       = HTTP2::VERSION
-  spec.authors       = ["Ilya Grigorik", "Kaoru Maeda"]
-  spec.email         = ["ilya@igvita.com"]
-  spec.description   = "Pure-ruby HTTP 2.0 protocol implementation"
+  spec.authors       = ['Ilya Grigorik', 'Kaoru Maeda']
+  spec.email         = ['ilya@igvita.com']
+  spec.description   = 'Pure-ruby HTTP 2.0 protocol implementation'
   spec.summary       = spec.description
-  spec.homepage      = "https://github.com/igrigorik/http-2"
-  spec.license       = "MIT"
+  spec.homepage      = 'https://github.com/igrigorik/http-2'
+  spec.license       = 'MIT'
 
-  spec.files         = `git ls-files`.split($/)
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ["lib"]
+  spec.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
+  spec.executables   = spec.files.grep(/^bin\//) { |f| File.basename(f) }
+  spec.test_files    = spec.files.grep(/^(test|spec|features)\//)
```

Comment:
Agreed - this can be fixed on the next rubocop release which will contain: https://github.com/bbatsov/rubocop/commit/e3eca69269fd9191a2315cdbd90adae98e32340b